ts-types/constructor/ensure.d.ts

```diff
@@ -0,0 +1,4 @@
+type EnsureFunction = (...args: any[]) => any;
```

Comment:
Is there a way to reuse types (e.g. import one from `ts-types/function/ensure.d.ts`) ?

Reply:
yes we can do that. Will update.